docs/guides/migration.md

```diff
@@ -0,0 +1,223 @@
+---
+subcategory: "Guides"
+page_title: "Migration"
+description: |-
+    An overview of resource and datasource migration changes
+---
+
+## Introduction
+
+Since the first release in September 2020, the Terraform ACI provider has come a long way, adding new resources and datasources to make managing Cisco ACI environments easier. Over the years, we've moved from SDKv1 to [SDKv2](https://developer.hashicorp.com/terraform/plugin/sdkv2), and now to the latest [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework), which is the recommended way to develop Terraform plugins and offers many advantages over SDKv2.
```

Comment:
Since the first release -> Since its first release

Reply:
datasources -> data sources ?

Reply:
changed everywhere